comps/guardrails/pii_detection/README.md

```diff
@@ -62,11 +79,18 @@ import requests
 import json
 
 proxies = {"http": ""}
-url = "http://localhost:6357/v1/dataprep"
-urls = [
-    "https://towardsdatascience.com/no-gpu-no-party-fine-tune-bert-for-sentiment-analysis-with-vertex-ai-custom-jobs-d8fc410e908b?source=rss----7f60cf5620c9---4"
+url = "http://localhost:6357/v1/piidetect"
+
+strategy = "ml"  # options: "ner", "ml"
+content = [
```

Comment:
It would be better to give a preview of the PII results?

Reply:
What kind of preview do you have in mind? The output is binary True or False.

Reply:
even if the output is a binary result, you'd better show it to the user and to let the user know what the the format of the result. Especially, i suppose you want to give the user an example use case here, you should let the user know what is the input and output like

Reply:
I added some descriptions in the readme. The idea is that the bigger application will use the True or False flag to go to different routes. @xuechendi and I will add other formats of input/output if there is customer request.